shared/common-adapters/user-proofs.desktop.js

```diff
@@ -51,6 +53,9 @@ class ProofRow extends Component<void, ProofRowProps, ProofRowState> {
       hovering: false,
       popupMenuPosition: {},
     }
+
+    this._onMouseEnter = () => this.setState({hovering: true})
```

Comment:
instead of making the closures on the render call we can just store it on ourselves